statefun-kafka-io/src/main/java/org/apache/flink/statefun/sdk/kafka/KafkaTopicPartition.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.flink.statefun.sdk.kafka;
+
+import java.util.Objects;
+
+/** Representation of a Kafka partition. */
+public final class KafkaTopicPartition {
+
+  private final String topic;
+  private final int partition;
+
+  public KafkaTopicPartition(String topic, int partition) {
+    this.topic = Objects.requireNonNull(topic);
+
+    if (partition < 0) {
+      throw new IllegalArgumentException(
+          "Invalid partition id: " + partition + "; value must be larger or equal to 0.");
+    }
+    this.partition = partition;
+  }
+
+  public String getTopic() {
```

Comment:
nit: can we stay consistent with the rest of the style in that package and omit `get` from the getters?

Reply:
will change